lib/alerts_viewer_web/components/core_components.ex

```diff
@@ -517,11 +517,13 @@ defmodule AlertsViewerWeb.CoreComponents do
 
   @doc ~S"""
   Renders a table with generic styling.
+  You can style rows by giving the row map a :row_class key with the classes as the value
+  Eg: rows |> Enum.map(& Map.put(&1, :row_class, "text-fuchsia-500 bg-orange-300"))
 
   ## Examples
 
       <.table id="users" rows={@users}>
-        <:col :let={user} label="id"><%= user.id %></:col>
+        <:col :let={user} label="id" title="hello world"><%= user.id %></:col>
```

Comment:
question: Did you mean to leave this title in there?

Reply:
I meant it as an example of how to put a title attribute on the column... maybe not very clear

Reply:
Oh shoot, I totally missed that it's in the documentation, my fault sorry.